man/systemd.netdev.xml

```diff
@@ -907,7 +907,7 @@
       <varlistentry>
         <term><varname>PeerTunnelId=</varname></term>
         <listitem>
-          <para>Specifies the peer tunnel id. Takes a number in the range 1—4294967295. The value used must
+          <para>Specifies the peer tunnel id. Takes a number in the range 1–4294967295. The value used must
```

Comment:
hmm, i was sure we use `…` for most cases like this, and not a minus or an mdash. I think the `…` syntax is the clearest since it cannot be mistaken as subtraction. i#d really try to unify on `…` (by which I mean the unicode ellipsis, not three full stop ASCII chars)

Reply:
I don't really care. I'm fine with "…" if we can use that everywhere consistently. Anyone disagrees?